variants/PRNTR_V2/variant.h

```diff
@@ -0,0 +1,194 @@
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *******************************************************************************
+*/
+
+#ifndef _VARIANT_ARDUINO_STM32_
+#define _VARIANT_ARDUINO_STM32_
+
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+/*----------------------------------------------------------------------------
+ *        Pins
+ *----------------------------------------------------------------------------*/
+
+#define PE0    0
+#define PE1    1
+#define PE2    2
+#define PE3    3
+#define PE4    4
+#define PE5    5
+#define PE6    6
+#define PE7    7
+#define PE8    8
+#define PE9    9
+#define PE10   10
+#define PE11   11
+#define PE12   12
+#define PE13   13
+#define PE14   14
+#define PE15   15
+
+#define PB0    16
+#define PB1    17
+#define PB2    18
+#define PB3    19
+#define PB4    20
+#define PB5    21
+#define PB6    22
+#define PB7    23
+#define PB8    24
+#define PB9    25
+#define PB10   26
+#define PB11   27
+#define PB12   28
+#define PB13   29
+#define PB14   30
+#define PB15   31
+
+#define PC0    32
+#define PC1    33
+#define PC2    34
+#define PC3    35
+#define PC4    36
+#define PC5    37
+#define PC6    38
+#define PC7    39
+#define PC8    40
+#define PC9    41
+#define PC10   42
+#define PC11   43
+#define PC12   44
+#define PC13   45
+#define PC14   46
+#define PC15   47
+
+#define PD0    48
+#define PD1    49
+#define PD2    50
+#define PD3    51
+#define PD4    52
+#define PD5    53
+#define PD6    54
+#define PD7    55
+#define PD8    56
+#define PD9    57
+#define PD10   58
+#define PD11   59
+#define PD12   60
+#define PD13   61
+#define PD14   62
+#define PD15   63
+
+#define PA0    64
+#define PA1    65
+#define PA2    66
+#define PA3    67
+#define PA4    68
+#define PA5    69
+#define PA6    70
+#define PA7    71
+#define PA8    72
+#define PA9    73
+#define PA10   74
+#define PA11   75
+#define PA12   76
+#define PA13   77
+#define PA14   78
+#define PA15   79
+
+// This must be a literal
+#define NUM_DIGITAL_PINS        80
+// This must be a literal with a value less than or equal to to MAX_ANALOG_INPUTS
+#define NUM_ANALOG_INPUTS       8
+#define NUM_ANALOG_FIRST        64
+
+// On-board LED pin number
+#define LED_BUILTIN             -1
```

Comment:
If no LED BUILTIN I guess you could simply comment or remove it.

Reply:
removed